packages/flutter_tools/lib/src/commands/upgrade.dart

```diff
@@ -28,7 +28,7 @@ class UpgradeCommand extends FlutterCommand {
       ..addFlag(
         'force',
         abbr: 'f',
-        help: 'Force upgrade the flutter branch, potentially discarding local changes.',
+        help: 'Force upgrade the Flutter branch, potentially discarding local changes.',
```

Comment:
Arguably `flutter` was correct here since it's the name of the repository? I could go either way.